### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by lfbrock**
- Add information about Native App Links
- Correct out of date Legal and Support link docs
- Update docs to reflect change in behaviour when a user is disabled using LDAP

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -909,6 +911,22 @@ Set an email for feedback or support requests.
 
 ________
 
+Native App Links
+```````````````````````````
+Mattermost Apps Download Page Link ``"AppDownloadLink": "https://about.mattermost.com/downloads/"`` 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Configurable link to a download page for Native Apps. When a link is present, an option to "Download Mattermost Apps" will be added in the Main Menu so users can find the download page. Leave this field blank to hide the option from the Main Menu. Defaults to a page on about.mattermost.com where users can download the iOS, Android, and Desktop clients. If you are using an `Enterprise App Store <https://docs.mattermost.com/deployment/push.html?highlight=enterprise%20app#push-notifications-and-mobile-devices>`_for your mobile apps, change this link to point to a customized download page where users can find the correct apps.
```
Comment (it33) [inline source/administration/config-settings.rst]: Need space after `_` for link to work

image

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -909,6 +911,22 @@ Set an email for feedback or support requests.
...
+
+Android App Download Link ``"AndroidAppDownloadLink": "https://about.mattermost.com/mattermost-android-app/"`` 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Configurable link to download the Android app. When a link is present, users who access the site on a mobile web browser will be prompted with a page giving them the option to download the app. Leave this field blank to prevent the page from appearing. If you are using an `Enterprise App Store <https://docs.mattermost.com/deployment/push.html?highlight=enterprise%20app#push-notifications-and-mobile-devices>`_for your mobile apps, change this link to point to the correct app.
```
Comment (it33) [inline source/administration/config-settings.rst]: Need space after `_` for link to work

image

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -909,6 +911,22 @@ Set an email for feedback or support requests.
...
+
+iOS App Download Link ``"IosAppDownloadLink": "https://about.mattermost.com/mattermost-ios-app/"`` 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Configurable link to download the iOS app. When a link is present, users who access the site on a mobile web browser will be prompted with a page giving them the option to download the app. Leave this field blank to prevent the page from appearing. If you are using an `Enterprise App Store <https://docs.mattermost.com/deployment/push.html?highlight=enterprise%20app#push-notifications-and-mobile-devices>`_for your mobile apps, change this link to point to the correct app.
```
Comment (it33) [inline source/administration/config-settings.rst]: Need space after `_` for link to work

**Comment by jasonblais** [comment]
Needs a rebase

**Comment by jasonblais** [comment]
Given the config-settings file was changed substantially, might be easiest to resubmit the changes on another PR.

esethna: Given lfbrock is out this week and you updated the config file, maybe have your help making the update?

**Comment by esethna** [comment]
Rebased and resubmitted here: https://github.com/mattermost/docs/pull/333

**Pull request closed without merging**